documentation/api/io.strimzi.api.kafka.model.KafkaAuthorizationCustom.adoc

```diff
@@ -0,0 +1,32 @@
+Custom authorization in Strimzi lets user configure their own `Authorizer` plugin to define the Access Control Lists (ACLs)
+
+ACLs allow you to define which users have access to which resources at a granular level.
+
+Configure the `Kafka` custom resource to use custom authorization.
+Set the `type` property in the `authorization` section to the value `custom`,
+and the set following properties.
```

Comment:
Can we add here something like ...

```
IMPORTANT: The custom authorizer needs to implement the `org.apache.kafka.server.authorizer.Authorizer` interface and support configuration of super-users from the `super.users` configuration field.
```

or something like that? This is important, because this is needed to configure the Strimzi compoenents.

Reply:
IMPORTANT: The custom `Authorizer` must implement the `org.apache.kafka.server.authorizer.Authorizer` interface, and support configuration of super users using the `super.users` configuration property.